SU2_CFD/src/numerics_direct_mean_inc.cpp

```diff
@@ -849,6 +981,79 @@ void CSourceIncBodyForce::ComputeResidual(su2double *val_residual, CConfig *conf
 
 }
 
+CSourceIncRotatingFrame_Flow::CSourceIncRotatingFrame_Flow(unsigned short val_nDim, unsigned short val_nVar, CConfig *config) : CNumerics(val_nDim, val_nVar, config) { 
+
+  Gamma = config->GetGamma(); 
+  Gamma_Minus_One = Gamma - 1.0; 
+
+} 
+
+CSourceIncRotatingFrame_Flow::~CSourceIncRotatingFrame_Flow(void) { } 
+
+void CSourceIncRotatingFrame_Flow::ComputeResidual(su2double *val_residual, su2double **val_Jacobian_i, CConfig *config) { 
+
+
+  unsigned short iDim, iVar, jVar; 
+  su2double Omega[3] = {0,0,0}, Momentum[3] = {0,0,0}, Velocity_i[3] = {0,0,0}; 
+
+  bool implicit     = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT); 
+
+  /*--- Retrieve the angular velocity vector from config. ---*/ 
+
+  for (iDim = 0; iDim < 3; iDim++){
+    Omega[iDim] = config->GetRotation_Rate(iDim)/config->GetOmega_Ref();
```

Comment:
You can put that in the constructor of the class

Reply:
Yep, done.